ci/dev/test-integration.sh

```diff
@@ -24,6 +24,13 @@ main() {
     path="$CODE_SERVER_PATH"
   fi
 
+  if [[ ! -f "test/integration/test-plugin/out/index.js" ]]; then
```

Comment:
Small optimization: only build test-plugin if not already built.

Reply:
Could this cause issues when developing?  For example making a change to the plugin but then it does not rebuild.

Actually I think we can skip this because `make` is supposed to avoid rebuilding if not necessary.

Reply:
Ah TIL! Cool, I'l remove it then.